### PR TITLE
Endpoints for sensor locations and sample data

### PR DESCRIPTION
**Pull request by AndrzejStarzyk**
I've added two endpoints for sensor locations and sample data based on files Metadane - stacje i stanowiska pomiarowe.xlsx and dane_aqm_2021_krk.xlsx from MsTeams. This was needed for displayg sensor locations on the frontend and couldn't find any appropriate endpoints.

### CODE REVIEW

File: src/main/resources/locations/stacje.csv
Comment (pgorgolew) [inline src/main/resources/locations/stacje.csv:1]: change to english

File: src/main/java/atmosfears/AtmosFearsBE/service/SampleSensorDataService.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            Set<AirParticulates> sampleData = new HashSet<>();
+            File source = new File(getClass().getClassLoader().getResource(LOCATIONS_DIR).toURI());
+            Set<String> paths = Stream.of(source.listFiles())
+                    .filter(file -> !file.isDirectory())
+                    .map(File::getPath)
+                    .collect(Collectors.toSet());
+
+            paths.forEach((String filePath) -> {
+                File dataFile = new File(filePath);
+                try (FileReader fr = new FileReader(dataFile);
```
Comment (pgorgolew) [inline src/main/java/atmosfears/AtmosFearsBE/service/SampleSensorDataService.java:36]: Is it only for testing, or will be used in final version? If yes, please use CSVReader or something like that - https://www.baeldung.com/java-csv-file-array#:~:text=4.-,OpenCSV,-We%20can%20address

File: src/main/java/atmosfears/AtmosFearsBE/service/LocationsService.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Service
+public class LocationsService {
```
Comment (pgorgolew) [inline src/main/java/atmosfears/AtmosFearsBE/service/LocationsService.java:17]: Same as in SampleSensorDataService

File: src/main/java/atmosfears/AtmosFearsBE/model/Location.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package atmosfears.AtmosFearsBE.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection = "locations")
```
Comment (pgorgolew) [inline src/main/java/atmosfears/AtmosFearsBE/model/Location.java:9]: We want to have it in database?

File: src/main/java/atmosfears/AtmosFearsBE/model/Location.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Location {
+    @Id
+    private String id;
+
+    private String code;
+    private Double N;
```
Comment (pgorgolew) [inline src/main/java/atmosfears/AtmosFearsBE/model/Location.java:18]: Tbh, I don't like this short field naming. What about changing it to (for example) `northCoordinate` and `eastCoordinate`

Comment (pgorgolew) [reply src/main/java/atmosfears/AtmosFearsBE/model/Location.java:18]: Also what is `code`?

**Comment by AndrzejStarzyk** [comment]
Thanks for feedback. I'm closing this PR since data I tried to mock here is now in the database.

**Pull request closed without merging**